clicommand/global.go

```diff
@@ -182,6 +182,18 @@ type APIConfig struct {
 	NoHTTP2          bool   `cli:"no-http2"`
 }
 
+// CacheConfig includes cache-related shared options for easy inclusion across
+// cache command config structs (via embedding).
+type CacheConfig struct {
+	Ids             string `cli:"ids"`
```

Comment:
Here's what I think:

```suggestion
	IDs             []string `cli:"ids"`
```

Reply:
done and moved to `clicommand/cache_shared.go`